packages/block-editor/src/components/link-control/index.js

```diff
@@ -33,7 +40,37 @@ import { focus } from '@wordpress/dom';
 import LinkControlSettingsDrawer from './settings-drawer';
 import LinkControlSearchItem from './search-item';
 import LinkControlSearchInput from './search-input';
+import LinkControlSearchCreate from './search-create-button';
+const CREATE_TYPE = '__CREATE__';
+let cancelableOnCreate;
+let cancelableCreateSuggestion;
```

Comment:
I don't think these can be in this scope, since the promises should be cancelled per instance of the component. There's nothing to say we couldn't have multiple `LinkControl`s rendered, each creating at the same time.

Reply:
Of course...and I expect this is why my tests are failing.

Reply:
It's not why my tests are failing 😢